### ISSUE TITLE
Wrong closure arity in map_key_acc/4 meta-predicate directive

### ISSUE THREAD

**Issue by pmoura** [vscosta/yap-6.3#41]
Wrong closure arity in `map_key_acc/4` meta-predicate directive (found by the Logtalk linter):

https://github.com/vscosta/yap-6.3/blob/13d88949c95ced7a84999a950ee37d4723e8a278/library/rbtrees.yap#L901

It should be:

```logtalk
:- meta_predicate map_key_acc(?,4,?,?).  % this is required.
```

**Comment by vscosta** [comment]
TRicky one, thanks!

**Comment by pmoura** [comment]
Fixed as of version YAP 6.5.0-6629c13c.
